Sources/DiscoverKit/View/RecommendationsView.swift

```diff
@@ -44,3 +54,14 @@ struct ListView: View {
         }
     }
 }
+
+extension View {
+    @ViewBuilder
+    func `if`<Content: View>(_ condition: Bool, modify: (Self) -> Content) -> some View {
+        if condition {
+            modify(self)
+        } else {
+            self
+        }
+    }
+}
```

Comment:
Hey @cyndichin , given the general format of this extension, it might be better suited for `MoSoCore` than `DiscoverKit`; what do you think?

Reply:
Since it's specific to View, do we think it might fit better with `DesignKit`?

Reply:
I updated it to be in `DesignKit`, but let me know if you prefer `MoSoCore` and happy to change it :)

Reply:
sure that works!!